src/query/combine.rs

```diff
@@ -2,7 +2,6 @@ use crate::{
     ColumnTrait, EntityTrait, IdenStatic, Iterable, QueryTrait, Select, SelectTwo, SelectTwoMany,
 };
 use core::marker::PhantomData;
-pub use sea_query::JoinType;
```

Comment:
can confirm this is already exported in `query.rs`

Reply:
Oh, it is not exported so I have reverted this change. [e4f4a12](https://github.com/SeaQL/sea-orm/pull/2056/commits/e4f4a129b9a1220ddc41fc5de2dff9866aaf9388)